tests/kernel/qa_test_klp.pm

```diff
@@ -30,7 +30,9 @@ sub run {
     # Set and check patch variables
     my $incident_id = get_var('INCIDENT_ID');
     my $patch       = get_var('INCIDENT_PATCH');
-    check_patch_variables($patch, $incident_id);
+    if (get_var('BETA')) {
+        check_patch_variables($patch, $incident_id);
+    }
```

Comment:
Is this commit necessary? Another commit in this PR also touches this part.

Reply:
Moreover, condition is wrong, this would exclude maintenance jobs without BETA. And another commit is fixing/changing it to exclude jobs with BETA. I see it quite confusing and believe that this commit is not correct.